adding_species/contributing.ipynb

```diff
@@ -0,0 +1,990 @@
+    "        695633.55941906,  695633.55941906,  695633.55941906,\n",
+    "        695633.55941906,  695633.55941906,  695633.55941906,\n",
+    "        695633.55941906,  695633.55941906,  695633.55941906,\n",
+    "        695633.55941906,  695633.55941906,  695633.55941906,\n",
+    "        695633.55941906,  695633.55941906,  695633.55941906,\n",
+    "        695633.55941906,  695633.55941906,  694677.60526595,\n",
+    "        694677.60526595,  694677.60526595,  694677.60526595,\n",
+    "        694677.60526595,  694677.60526595,  694677.60526595,\n",
+    "        694677.60526595,  694677.60526595,  694677.60526595,\n",
+    "        694677.60526595,  694677.60526595,  693429.24269629,\n",
+    "        693429.24269629,  693429.24269629,  693429.24269629,\n",
+    "        693429.24269629,  693429.24269629,  692716.93526192,\n",
+    "        692716.93526192,  692379.41939462,  692379.41939462,\n",
+    "        690176.6983718 ,  690176.6983718 ,  688681.83478372,\n",
+    "        688681.83478372,  687162.86783629,  687162.86783629,\n",
+    "        686235.36369922,  686235.36369922,  686150.90910499,\n",
+    "        686150.90910499,  685951.34145025,  685951.34145025,\n",
+    "        685005.95081601,  685005.95081601,  684576.59330447,\n",
+    "        684576.59330447,  684576.59330447,  684576.59330447,\n",
+    "        683889.43627665,  683889.43627665,  683889.43627665,\n",
+    "        683889.43627665,  683889.43627665,  683889.43627665,\n",
+    "        683889.43627665,  683889.43627665,  687172.58638815,\n",
+    "        687172.58638815, 1980227.21562606, 1980227.21562606,\n",
+    "       1989715.58702617, 1989715.58702617, 1992430.35210815,\n",
+    "       1992430.35210815, 1998953.44050118, 1998953.44050118,\n",
+    "       1998953.44050118, 1998953.44050118, 1998160.88128216,\n",
+    "       1998160.88128216, 1998160.88128216, 1998160.88128216,\n",
+    "       1998160.88128216, 1998160.88128216, 1972764.03439268,\n",
+    "       1972764.03439268, 1903551.76041575, 1903551.76041575,\n",
+    "        543762.2932594 ,  543762.2932594 ,  530289.21416049,\n",
+    "        530289.21416049,  624896.67937685,  624896.67937685,\n",
+    "       1680052.006021  , 1680052.006021  ,  322144.56539297,\n",
+    "        322144.56539297,   77334.77862527,   77334.77862527,\n",
+    "        409527.12801416,  409527.12801416]\n",
+    "        )\n",
+    "\n",
+    "    demographic_events = []\n",
+    "    for sz, t in zip(sizes, times):\n",
+    "        demographic_events.append(\n",
+    "            msprime.PopulationParametersChange(time=t, initial_size=sz, population_id=0)\n",
+    "        )\n",
+    "    populations = [\n",
+    "        stdpopsim.Population(\n",
+    "            id=\"GAS\",\n",
+    "            description=\"Gabon gabiae population\",\n",
+    "            )\n",
+    "        ]\n",
+    "\n",
+    "    return stdpopsim.DemographicModel(\n",
```

Comment:
It would be better to use PiecewiseConstantSize, wouldn't it? This is going to be super confusing because the catalog probably won't contain models that look like this (with demographic_events, etc) for much longer.

Reply:
right, i agree it would be better to use PiecewiseConstantSize but I came across two pedagogical reasons why i didn't.

the first was that to do this correctly I would use inheritance (at least how i started to code it) and I think i'd lose a lot of people right there. the second is that currently most of the models in the catalog are written in this old style, so when people go looking for examples this is what they'll see.

Reply:
OK, fair enough.